content/tugboat-cli/install-the-cli.md

```diff
@@ -38,6 +38,25 @@ Extract the downloaded `tugboat.tar.gz` file to `/usr/local/bin`
 tar -zxf tugboat.tar.gz -C /usr/local/bin/
 ```
 
+#### MacOS Catalina
+
+With Catalina's enhanced security features, the first time you try to execute a
+CLI command, you'll get a dialog telling you that the Tugboat CLI can't be
+opened:
+
+![Tugboat CLI can't be opened](../../_images/tugboat-cli-cant-be-opened.png)
+
+Until we're able to release a signed version of the CLI, here's how to
+workaround this issue:
```

Comment:
Minor question for my own edification: should this be "work around"? I guess I was thinking that you "work around" (verb) something, resulting in a "workaround" (noun).

Reply:
Good catch, @q0rban - it should be the verb version here; I rewrote my original sentence but forgot to update from noun to verb. Need more coffee...